.drone.yml

```diff
@@ -1,5 +1,9 @@
+env:
+- REPORT_CARD_GITHUB_STATUS_TOKEN=$$report_card_github_status_token
+- REPORT_CARD_GITHUB_REPO_TOKEN=$$report_card_github_repo_token
 image: node0.10
 script:
+- sudo pip install -q git+https://$REPORT_CARD_GITHUB_REPO_TOKEN@github.com/Clever/report-card.git; GITHUB_API_TOKEN=$REPORT_CARD_GITHUB_STATUS_TOKEN report-card --publish || true
```

Comment:
This introduced a bug running drone tests... fixed here https://github.com/Clever/clever-js/pull/40